### ISSUE TITLE
Using -test suffix for autogenerated test configuration is not a good idea.

### ISSUE THREAD

**Issue by mnd999** [scalacenter/bloop-maven-plugin#62]
### Describe the bug

Say we have a Maven project with two modules, `module` and `module-test`. Assume they both have code under `src/test/scala` and `src/main/scala`.  

The generated bloop config for the code under `module/src/test/scala` will conflict with the one for `module-test/src/main/scala` because test code gets an autogenerated `-test` suffix and code under main does not get any suffix.

Either we should use a less commonly used suffix for test modules, or we should append a suffix (e.g. -compile) for both.

### Expected behavior

Config under .bloop should be generated for all 4 compilation units. Instead we only get 3 as one overwrites the other. 

### Operating system

Linux

### Version of Maven

6.4.10-arch1-1

### Version of bloop-maven-plugin

1.5.4

### Version of Bloop

Didn't get that far

**Comment by ckipp01** [comment]
So this would probably need to change not just in the plugin, but also the entire bloop ecosystem, which might have some larger ramifications. @tgodzik, has this come up before?

**Comment by tgodzik** [comment]
I don't think it popped up before, but it's not a huge change, different build tools generate different module names. We would need to find conflicts and maybe add a fallback name in that case. Like `module-test-scope`
